src/common/FundFilter/FundFilter.js

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 import { SelectionFilter } from '@folio/stripes-acq-components';
-
-import {
-  getFundOptions,
-} from '../utils';
-import { fundsShape } from '../shapes';
+import { fundShape } from '../shapes';
```

Comment:
I don't get this change, could you please provide a reason?

Reply:
It happened due to Eslint asked to move line `import PropTypes from 'prop-types';` above than other imports.

Reply:
Not actual after the last commit.